easyconfigs/v/ViennaRNA/ViennaRNA-2.6.4-foss-2022b.eb

```diff
@@ -0,0 +1,55 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'ConfigureMake'
+
+name = 'ViennaRNA'
+version = '2.6.4'
+
+homepage = 'https://www.tbi.univie.ac.at/RNA/'
+description = """The Vienna RNA Package consists of a C code library and several
+stand-alone programs for the prediction and comparison of RNA secondary structures."""
+
+toolchain = {'name': 'foss', 'version': '2022b'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://www.tbi.univie.ac.at/RNA/download/sourcecode/%(version_major)s_%(version_minor)s_x/']
+sources = [SOURCE_TAR_GZ]
+
+builddependencies = [
+    ('Autotools', '20220317'),
+    ('flex', '2.6.4'),
+    ('pkg-config', '0.29.2'),
+    ('Bison', '3.8.2'),
+    ('SWIG', '4.1.1'),
+]
+
+dependencies = [
+    ('Python', '3.10.8'),
+    ('Perl', '5.36.0'),
+    ('GSL', '2.7'),
+    ('libgd', '2.3.3'),
+    ('X11', '20221110'),
+]
+
+preconfigopts = r"sed -e 's/-llapack/\$LIBLAPACK/g' -i ./src/RNAxplorer/configure && "
```

Comment:
Is this the right approach?

Reply:
Yes, as it does not look to be possible to control it directly.